lib/cc/analyzer/issue_validations/other_locations_format_validation.rb

```diff
@@ -25,7 +30,8 @@ def message
         private
 
         def other_location_valid?(location)
-          CHECKS.all? do |klass|
+          path = location && location["path"]
+          @location[path] ||= CHECKS.all? do |klass|
```

Comment:
Can you rename `@location` to `@other_locations`? This would differentiate it from the local `location` variable, and indicate it's a Hash of many location values, not just one.